backend/main/server/app.go

```diff
@@ -589,10 +589,10 @@ func (a *App) createVoteForProposal(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	// validate user signature
-	if err := a.FlowAdapter.UserTransactionValidate(v.Addr, v.Message, v.Composite_signatures, v.TransactionId, a.TxOptionsAddresses, p.Choices); err != nil {
```

Comment:
we will want to resolve the issue here before merging because otherwise Ledger wallets won't be validating properly